src/System Application/App/Extension Management/src/ExtensionManagement.Page.al

```diff
@@ -208,37 +216,15 @@ page 2500 "Extension Management"
                         CurrPage.Update(false);
                     end;
                 }
-                action("Extension Marketplace")
-                {
-                    Caption = 'Extension Marketplace';
-                    Enabled = IsSaaS;
-                    Image = NewItem;
-                    ToolTip = 'Browse the extension marketplace for new extensions to install.';
-                    Visible = not IsOnPremDisplay;
-#if not CLEAN24                    
-#pragma warning disable AL0432
-                    RunObject = page "Extension Marketplace";
-#pragma warning restore AL0432
-#else                    
-                    trigger OnAction()
-                    begin
-                        Hyperlink('https://aka.ms/bcappsource');
-                    end;
-#endif
-                }
                 action("Microsoft AppSource Gallery")
                 {
                     Caption = 'AppSource Gallery';
                     Enabled = IsSaaS;
                     Image = NewItem;
                     ToolTip = 'Browse the Microsoft AppSource Gallery for new extensions to install.';
                     Visible = not IsOnPremDisplay;
+                    RunObject = Page 2515;
```

Comment:
What about adding a comment here to explain why the page ID instead of the name is used? See also
https://github.com/microsoft/BCApps/pull/1435/files#r1660628195

Reply:
Not a fan but there we go :)

Reply:
@SBalslev Resolved but still unchanged? ;-)
```suggestion
                    RunObject = Page 2515; // Using ID to avoid circular dependency
```

Reply:
wierd - let me try again

Reply:
I've seen this happening over and over again - requests are marked as "Resolved", but the code was not updated in fact.
Maybe some GitHub glitch that should be paid attention to?